nemo/collections/nlp/modules/common/hyena/fftconv_wrapper.py

```diff
@@ -0,0 +1,129 @@
+import math
+
+import torch
+from einops import rearrange
+from fftconv import fftconv_bwd, fftconv_fwd
```

Comment:
I wonder if this should be wrapped in try/catch exception block for people who don't have fftconv installed?